Scripts/utils/freight_costs.py

```diff
@@ -0,0 +1,64 @@
+import numpy
+from typing import Dict
+
+
+def calc_road_cost(unit_cost_parameters: Dict[str, Dict],
+                   distance: numpy.ndarray,
+                   time: numpy.ndarray,
+                   toll_cost: numpy.ndarray = 0) -> numpy.ndarray:
+    mode_cost = {}
+    for mode, parameters in unit_cost_parameters.items():
+        time_cost = time * parameters["time_based_cost"]
+        dist_cost = distance * parameters["distance_based_cost"]
```

Comment:
Using the same logic as elsewhere, you could consider having matrices in a dict with keys "time" and "dist" and parameters named exactly in the same way, so we do not need to have the keys hard-coded here.

Reply:
Will refactor cost calculations and for all freight modes and their unit cost parameter json files to improve readability.